---
layout: about
permalink: /
title: <strong>Shengwei</strong> Hou
description: Dr. rer. nat. ( Ph.D.), Bioinformatician & Computational Biologist # <a href="">Affiliations</a>. Address. Contacts. Moto. Etc.

profile:
  align: right
  image: prof_pic.jpg
  address: >
    <p style="font-size:14px;color:rgba(47,86,135,1);font-weight:bold;">Address:</p>
    <p style="font-size:10px;color:rgba(47,86,135,1);">AHF230</p>
    <p style="font-size:10px;color:rgba(47,86,135,1);">3616 Trousdale Pkwy</p>
    <p style="font-size:10px;color:rgba(47,86,135,1);">University of Southern California</p>
    <p style="font-size:10px;color:rgba(47,86,135,1);">Los Angeles, CA 90089</p>

news: true
social: true
---

Welcome to my website.

I'm currently doing my postdoc study collaborating with Prof. [Jed A. Fuhrman](https://dornsife.usc.edu/labs/fuhrmanlab) at *University of Southern California* (USC). Before I moved here in April 2018, I finished my Ph.D. and a short-term postdoc study in [Cyanolab](http://www.cyanolab.de/) supervised by Prof. [Wolfgang R. Hess](https://scholar.google.de/citations?user=rKJIEk8AAAAJ&hl=en) at *University of Freiburg* in Germany, where I had a curiosity-oriented systematic training and professional development in Computational Biology and Bioinformatics, with a focus on the non-coding elements encoded in three domains of life. Ahead of that, I got a master degree in *Genetics* at *Institute of Hydrobiology, Chinese Academy of Sciences* ([IHB](http://english.ihb.cas.cn/)), supervised by Prof. [Xudong Xu](http://sourcedb.ihb.cas.cn/en/expert/pl/200907/t20090722_2156195.html) on a project of understanding the regulatory roles of genes related to heterocyst development and pattern formation in the multicellular prokaryotic model organism *Anabaena* sp. PCC 7120. Please check [vitae]({{ site.baseurl }}/vitae/) for my detailed curriculum vitae.

I'm currently involved in the "Simons Collaboration on Computational Biogeochemical Modeling of Marine Ecosystems ([CBIOMES](https://cbiomes.org/about/))" project led by Prof. [Michael J. Follows](https://www.simonsfoundation.org/team/michael-j-follows/). The CBIOMES project aims to develop mathematical models to quantitatively understand the biogeographical patterns, structures and interactions of diverse marine microbial communities at different space and time scales, and to model the essential traits of marine microbes and their contribution to the global elemental cycles. The [Fuhrman Lab](https://dornsife.usc.edu/labs/fuhrmanlab) is one of the core members of this multidisciplinary collaboration, with a focus on providing marine microbial genetic and genomic information for the modelling purposes. Specifically, the sub-project for me is using genome-resolved metagenomic and metatranscriptomic approaches to understand the distribution and succession of diverse marine microbes and their genetic properties in the context of, but not limited to the "San Pedro Ocean Time-series ([SPOT](https://dornsife.usc.edu/spot/))". 

As shown in the following figure (<a href="#CBIOMES_Workflow">Figure 1</a>), briefly, we 

1. collect seawater samples from SPOT and the coast of Southern California,
2. enrich microbial materials by filtering the seawater through filters,
3. extract the whole community DNA and RNA from the filters, 
4. construct metagenomic and metatranscriptomic libraries for high-throughput sequencing, 
5. generate assemblies from quality-controlled reads,
6. bin assembled contigs into Metagenome Assembled Genomes (MAGs), 
7. classify microbes into functional categories based on their genetic and transcriptional properties,
8. collaborate with modellers to predict their global distribution and contribution to elemental cycles.

<div class="col three center"  id="CBIOMES_Workflow">
    <img class="col three center" src="{{ site.baseurl }}/assets/img/CBIOMES_Project_Overview.png" style="width:200px;height:500px;" alt="" title="project overview"/>
</div>

<div class="col three caption">
    Figure 1: A schematic representation of the sample collection and data analysis workflow
</div>